dojo/unittests/tools/test_wpscan_parser.py

```diff
@@ -59,7 +59,7 @@ def test_parse_file_with_multiple_vuln_has_multiple_finding(self):
         for finding in findings:
             for endpoint in finding.unsaved_endpoints:
                 endpoint.clean()
-        self.assertEqual(8, len(findings))
+        self.assertEqual(11, len(findings))
         finding = findings[0]
         self.assertEqual("8873", finding.unique_id_from_tool)
         self.assertNotEqual("Info", finding.severity)  # it is a vulnerability so not 'Info'
```

Comment:
Please add more checks to test new findings. You did a good job to create new data but we need to test them in the unit tests.